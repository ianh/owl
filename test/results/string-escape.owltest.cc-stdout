a (0 - 43)
  string - oo
 (0 - 7)
  string - oo
oooo (8 - 21)
  string - oo
ooooiiaauu (22 - 43)
