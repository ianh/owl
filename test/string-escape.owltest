'boo\n'
"boo\nbooboo"
"boo\nbooboobiibaauu"
---a = string*
