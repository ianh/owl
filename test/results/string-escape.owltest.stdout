. 'boo\n'
  a------

. "boo\nbooboo"
  -------------

. "boo\nbooboobiibaauu"
  ---------------------
